include/seastar/rpc/rpc.hh

```diff
@@ -519,6 +519,9 @@ public:
             c->_parent = this->weak_from_this();
             c->_is_stream = true;
             return c->await_connection().then([c, this] {
+                if (_error) {
+                    throw std::runtime_error("Parent socket is closing");
```

Comment:
Shouldn't we throw `rpc::closed_error` instead?

Reply:
Yes, why not. Fixed in v2